waspc/tools/ensure_todoapp_builds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
```

Comment:
You put this in separate file so we can also run it locally?

Reply:
We could, but just easier to manage this way mainly.